### DISCUSSION TITLE
Feature Request: GitHub Rendering Support for Pluto.jl Scripts

### DISCUSSION THREAD

**Discussion by alvgaona** [community/community#110687] [Other Feature Feedback, Questions, & Ideas]
### Topic Area

Product Feedback

### Description

As an avid user of Pluto.jl for interactive and exploratory data analysis and scientific computing, I often find myself sharing my notebooks and scripts on GitHub for collaboration and version control. However, unlike Jupyter notebooks, GitHub does not currently support rendering Pluto.jl scripts directly in the browser, which limits the accessibility and usability of these files for collaborators and visitors.

Pluto.jl offers a dynamic and interactive environment for coding in Julia, enabling users to create living documents with reactive elements that update in real-time. It would be incredibly beneficial for the Pluto.jl community if GitHub could support the rendering of Pluto.jl scripts in a similar manner to Jupyter notebooks.

By enabling GitHub rendering support for Pluto.jl scripts, users would be able to:

1. Seamlessly view and interact with Pluto.jl notebooks directly within the GitHub interface.
1. Collaborate more effectively by providing feedback, suggestions, and modifications directly on Pluto.jl scripts.
1. Explore and understand Pluto.jl scripts without needing to download or run them locally.
1. I propose that GitHub adds native support for rendering Pluto.jl scripts, similar to its existing support for Jupyter notebooks. This feature would enhance the accessibility and visibility of Pluto.jl projects hosted on GitHub, ultimately fostering greater collaboration and innovation within the Julia community.

If you agree that adding GitHub rendering support for Pluto.jl scripts would be valuable, please consider adding your voice to this discussion by sharing your thoughts, use cases, and any additional suggestions.

**Comment by AtilaSaraiva** [comment]
I really want this feature too!
